### PR TITLE
Fix issues introduced with Owncloud 9.0

### PR DESCRIPTION
**Pull request by drybjed**

### CODE REVIEW

**Comment by ypid** [comment]
:+1: Nice that we don’t need to go [this path](https://github.com/debops/ansible-owncloud/issues/37). Closes https://github.com/debops/ansible-owncloud/issues/37

**Comment by drybjed** [comment]
@ypid Well, not so fast, I had errors during installation finish, but let's see how Travis does. In any case, I'll probably merge this patch even with errors since otherwise role is unusable, we can debug the autoconfiguration later.

**Comment by drybjed** [comment]
OK, this error was unexpected... I wonder if ownCloud packages for Debian and Ubuntu differ now, and how big the differences are, even between releases (Precise vs Jessie). Let me try to peek at the Travis CI build to see the changes.

File: defaults/main.yml [hunk 1]
```diff
@@ -41,7 +41,7 @@ owncloud_edition: 'community'
 #
 # ownCloud version in the format ``major_release.minor_release`` or ``testing``.
 # See http://download.opensuse.org/repositories/isv:/ownCloud:/community:/ for the full list.
-owncloud_release: 8.1
+owncloud_release: 9.0
```
Comment (ypid) [inline defaults/main.yml:44]: Shouldn't we explicitly set that to a string?

Comment (drybjed) [inline defaults/main.yml:44]: I'm not sure if that variable is even needed anymore... It's used in 1 place to check if version is 8.0, but it's not autodetected.

**Comment by drybjed** [comment]
So, after a quick look at [Debian repository](https://download.owncloud.org/download/repositories/stable/Debian_8.0/) vs [Ubuntu repository](https://download.owncloud.org/download/repositories/stable/Ubuntu_12.04/)... Wat... Are these packages still being built? Ubuntu 14.04 [has all the packages](https://download.owncloud.org/download/repositories/stable/Ubuntu_14.04/). I suppose that custom repository selection code is needed.

**Comment by drybjed** [comment]
OK, I'm going to merge these changes for now, but more work is needed to make it usable.

**Pull request merged**